Form-Controls/index.html

```diff
@@ -13,15 +13,70 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <div>
+          <label for="fname">First name</label>
+          <input type="text" name="fname" id="fname" required>
+        </div>
+      </br>
+        <div>
+          <label for="lname">Last name</label>
+          <input type="text" name="lname" id="lname" required>
+        </div>
+      </br>
+        <div>
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" required>
+        </div>
+        <div>
+          <h3>What colour of a t-shirt do you want?</h3>
+        <div>
+          <label for="colour">Purple</label>
+          <input type="radio" name="colour" value="purple">
+        </div>
+        <div>
+          <label for="colour">Black</label>
+          <input type="radio" name="colour" value="black">
+        </div>
+        <label for="colour">Green</label>
+          <input type="radio" name="colour" value="green">
+      </div>
+      <div>
+        <h3>Preferable size</h3>
+        <div>
+            <label for="xs">XS</label>
+            <input type="radio" name="size" value="xs">
+        </div>
+        <div>
+            <label for="s">S</label>
+            <input type="radio" name="size" value="s">
+        </div>
+        <div>
+            <label for="m">M</label>
+            <input type="radio" name="size" value="m">
+        </div>
+        <div>
+          <label for="l">L</label>
+          <input type="radio" name="size" value="l">
+      </div>
+      <div>
+        <label for="xl">XL</label>
+        <input type="radio" name="size" value="xl">
+    </div>
+    <div>
+      <label for="xxl">XXL</label>
+      <input type="radio" name="size" id="value">
+  </div>
+    </div>
+      <div>
+        <label for="qnt">Quantity</label>
+        <input type="number" name="qnt" id="qnt" min="1" max="20" required>
+    </div>
+  </br>
+    <button type="submit">Submit</button>
       </form>
     </main>
     <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
+      <h2>Elizabeth Tabala</h2>
```

Comment:
Please change heading level 2 to other tag here. Your name in the footer is not semantically a header, but something else.